sys-apps/systemd-utils/systemd-utils-254.3.ebuild

```diff
@@ -340,8 +340,6 @@ multilib_src_compile() {
 				targets+=(
 					# Used by udev-test.pl
 					systemd-detect-virt
-					test/sys
-					test-udev
```

Comment:
Any replacements for these / new tests to add?

Reply:
Addressed in 8291739b1a578f93836f9045246e047c69281c18.